core/trino-main/src/main/java/io/trino/execution/QueryManagerConfig.java

```diff
@@ -351,6 +353,18 @@ public QueryManagerConfig setQueryExecutionPolicy(String queryExecutionPolicy)
         return this;
     }
 
+    public boolean isHideUnaccessibleColumns()
+    {
+        return hideUnaccessibleColumns;
+    }
+
+    @Config("query.hide-inaccessible-columns")
```

Comment:
`@ConfigDescription`

Reply:
`@Config("query.filter-restricted-columns")`?

Reply:
i like "hide" more.

also "restricted" could be understood as referring to inaccessible & masked columns, whereas we don't want to hide masked columns